frontend/src/pages/ChainMemberList.tsx

```diff
@@ -460,6 +471,9 @@ function ParticipantsTable(props: {
                         {SizeBadges(t, u.sizes)}
                       </span>
                     </td>
+                    <td className="text-center">
+                      {signedUpOn(u.chains.map((c) => c.created_at))}
```

Comment:
`u.chains` gives a list of connected chains related to the current user. 
Filter to the object to the chain of the current page